lib/utils/getDeviceInfo.js

```diff
@@ -41,12 +41,19 @@ async function getDevicesDetails(devicesIds) {
  * @param {boolean} short
  */
 function getDeviceName(deviceInfo, short = false) {
-	const {manufacturer, model, customName} = deviceInfo;
+	const {manufacturer, model, customName, deviceId} = deviceInfo;
+
+	let name;
 
 	if (customName)
-		return short ? customName : `${customName} (${manufacturer} ${model})`;
+		name = short ? customName : `${customName} (${manufacturer} ${model})`;
```

Comment:
As I remember if customName not specified this field will be filled with manufacturer and model by default?